scripts/makepbcore.py

```diff
@@ -886,6 +893,9 @@ def main(args_):
         ififuncs.sha512_update(sha512_manifest, csv_filename)
         print((' - Updating %s with %s' % (sha512_manifest, csv_filename)))
         print(metadata_error)
+    if failed_path:
```

Comment:
I'd suggest adding this to the log as well so that there's a record there. There should be other functions that show how the logs are used and you can mimic the syntax usually used.

Reply:
Thanks! I will have a try on this now.